.storybook/static/healthcare-theme.css

```diff
@@ -62,7 +62,7 @@
 --color-secondary-dark: #107b0d;
 --color-secondary-darker: #0d600a;
 --color-secondary-darkest: #094507;
---color-secondary-light: #0d600a;
+--color-secondary-light: #59ac56;
```

Comment:
This was an artifact of merging epic/css-reset into this branch

Reply:
You could merge main directly into the epic branch to get rid of this